packages/drivers/iframe-driver/src/outerDocumentServiceFactory.ts

```diff
@@ -208,7 +208,7 @@ export class DocumentServiceFactoryProxy implements IDocumentServiceFactoryProxy
             clientId: deltaStream.clientId,
             existing: deltaStream.existing,
             get initialClients() { return deltaStream.initialClients; },
-            get initialContents() { return deltaStream.initialContents; },
+            get initialContents() { return [] as IContentMessage[]; },
```

Comment:
Why not delete it from here?

Reply:
Because I need to build an updated server package and publish that first so the IConnected interface doesn't require initialContents